pages/linux/toolbox-rm.md

```diff
@@ -0,0 +1,16 @@
+# toolbox rm
+
+> Remove one or more `toolbox` containers.
+> More information: <https://manned.org/toolbox-rm.1>.
+
+- Remove a toolbox container:
+
+`toolbox rm {{container_name}}`
+
+- Remove all `toolbox` containers:
+
+`toolbox rm --all`
+
+- Force the removal of all currently active `toolbox` containers:
+
+`toolbox rm --force`
```

Comment:
From the man page it sounds more like the command should look like this:
```suggestion
- Force the removal of a currently active `toolbox` container:

`toolbox rm --force {{container_name}}`
```
Is this correct?

Reply:
Yes, It is correct